UTM.xcodeproj/project.pbxproj

```diff
@@ -1833,7 +1918,7 @@
 			buildSettings = {
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				CODE_SIGN_STYLE = Automatic;
-				DEVELOPMENT_TEAM = "";
+				DEVELOPMENT_TEAM = JE2494MFB5;
```

Comment:
I wish there's a way for Xcode to not put this into the project file.

Reply:
xcconfig?

Reply:
TIL. I will add this in the future.